### PR TITLE
Can climb and changed shooting range to enum

### PR DESCRIPTION
**Pull request by BOSSkabab**

### CODE REVIEW

File: lib/views/mobile/screens/pit_view/pit_toggle.dart [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        selectedColor: Colors.blue,
+        selectedBorderColor: Colors.blue,
+        children: <Widget>[
+          Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 10),
+            child: Text(title),
+          ),
+        ],
+        isSelected: isSelected,
+        onPressed: (final int i) => setState,
```
Comment (itamarsch) [inline lib/views/mobile/screens/pit_view/pit_toggle.dart:26]: You don't call the function here this widget does nothing

File: lib/views/mobile/screens/pit_view/pit_toggle.dart [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+class PitToggle extends StatelessWidget {
+  const PitToggle({
+    super.key,
+    required this.setState,
+    required this.isSelected,
+    required this.title,
+  });
+  final void Function() setState;
+  final List<bool> isSelected;
```
Comment (itamarsch) [inline lib/views/mobile/screens/pit_view/pit_toggle.dart:11]: Make selected a bool

File: lib/views/mobile/screens/pit_view/pit_toggle.dart [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import "package:flutter/material.dart";
+
+class PitToggle extends StatelessWidget {
+  const PitToggle({
+    super.key,
+    required this.setState,
+    required this.isSelected,
+    required this.title,
+  });
+  final void Function() setState;
```
Comment (itamarsch) [inline lib/views/mobile/screens/pit_view/pit_toggle.dart:10]: Call this onPressed instead

**Review by itamarsch — CHANGES_REQUESTED** [review]

File: lib/views/mobile/screens/pit_view/pit_toggle.dart (hunk 1 above)
Comment (BOSSkabab) [reply lib/views/mobile/screens/pit_view/pit_toggle.dart:26]: but i don't use i in the setState

Comment (BOSSkabab) [reply lib/views/mobile/screens/pit_view/pit_toggle.dart:26]: but i don't use i in the setState

**Pull request merged**